pkgs/applications/misc/postage/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, runCommand, postgresql, openssl } :
+
+stdenv.mkDerivation rec {
+  name = "postage-${version}";
+  version = "HEAD";
```

Comment:
Note that `(parseDrvName "postage-HEAD").version == ""`, i.e., it's an invalid version string.